### PR TITLE
fix: wrong font style in input fields of Billing Address screen

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Bugfix

This PR resolves [CX-1911]

### Description

|Palette V3|Palette V2|
|---|---|
|image|image|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Change input field font style and capitalize label - rajsam003

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1911]: https://artsyproduct.atlassian.net/browse/CX-1911

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Change input field font style and capitalize label - rajsam003)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 446885ce357c863293d6b2a6e680837ae9169f31
</p>

File: src/lib/Components/Bidding/Screens/BillingAddress.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ interface StyledInputProps extends InputProps {
 const StyledInput: React.FC<StyledInputProps> = ({ label, errorMessage, onLayout, ...props }) => (
   <Flex mb={4} onLayout={onLayout}>
     <Serif size="3" mb={2}>
-      {label}
+      {label?.toUpperCase()}
```
Comment (olerichter00) [inline src/lib/Components/Bidding/Screens/BillingAddress.tsx:36]: I think we can remove this line. The `Input` component will be migrated to V3 with this PR https://github.com/artsy/eigen/pull/5391 and for now we can keep the title as it is.

File: src/lib/Components/Bidding/Elements/TextInput.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export interface TextInputProps
     RNTextInputProps {}
 
 export const TextInput = styled.TextInput.attrs<TextInputProps>({})`
-  font-family: "${Fonts.GaramondRegular}";
+  font-family: "${Fonts.Unica77LLRegular}";
```
Comment (olerichter00) [inline src/lib/Components/Bidding/Elements/TextInput.tsx:28]: Just removing the font-family here should be enough, I guess.

**Review by olerichter00 — COMMENTED** [review]
Thanks for working on this one. The space between the title and the actual input seem to be different (looking at the screenshot).

**Comment by rajsam003** [comment]
> Thanks for working on this one. The space between the title and the actual input seem to be different (looking at the screenshot).

**Comment by rajsam003** [comment]
> The space between the title and the actual input seem to be different (looking at the screenshot).

@olerichter00 It's not in the scope of the task. So I didn't touch it. Should I work on it with the same ticket or create a sub ticket?

**Comment by olerichter00** [comment]
@rajsam003 Yes, I think we should address that and try to align the space below the title.

**Review by olerichter00 — APPROVED** [review]
Looks good! 🚀

**Comment by olerichter00** [comment]
#awesome

**Pull request merged**